code/src/lib/Card.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export const Card = ({ title, secondaryText, coverImage, className, children }) => {
+
+  return (
+    <Container className={className}>
+      <TitleBar>
+        <div>
+          {coverImage && <CoverImage src={coverImage} />}
+          {title && <Title>{title}</Title>}
+          {secondaryText && <SecondaryText>{secondaryText}</SecondaryText>}
+        </div>
+      </TitleBar>
+      <Content>
+        {children && <ChildrenContent>{children}</ChildrenContent>}
+      </Content>
+    </Container>
+  )
+}
```

Comment:
Is your plan to use this Card instead of the paragraphs in ScannedInfo.js? In that case you can do something like this. When you mount Card in ScannedInfo.js, you can send the scannedItem as props to the Card-component and use it to display the relevant info there. So in ScannedInfo.js:

`<Card scannedItem={scannedItem.product} />`

and then here in Card.js you can pick it up and fill the card with whatever content you like:
```
export const Card = ({ scannedItem }) => {
  return (
    <Container>
      <TitleBar>
        <div>
          {scannedItem.image_url && <CoverImage src={scannedItem.image_url} />}
          {scannedItem.product_name && <Title>{scannedItem.product_name}</Title>}
        </div>
      </TitleBar>
      <Content>
        {scannedItem.origins && <ChildrenContent>{scannedItem.origins}</ChildrenContent>}
      </Content>
    </Container>
  )
}
```